src/fileoverview_comment_transformer.ts

```diff
@@ -28,6 +28,43 @@ export function isClosureFileoverviewComment(text: string) {
   return current !== null && current.tags.some(t => FILEOVERVIEW_COMMENT_MARKERS.has(t.tagName));
 }
 
+/**
+ * Given a parsed @fileoverview comment, ensures it has all the attributes we need.
+ * This function can be called to modify an existing comment or to make a new one.
+ *
+ * @param tags Comment as parsed list of tags; modified in-place.
+ */
+function modifyFileoverviewComments(tags: jsdoc.Tag[]) {
+  // Ensure we start with a @fileoverview.
```

Comment:
FWIW the order I think doesn't matter, but this is cleaner.

Reply:
This also kept the test churn down (it would put the @fileoverview second on every test otherwise).